tabulator/__init__.py

```diff
@@ -3,10 +3,22 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
+import io
+import os
+
+
+# General
 
 from .stream import Stream
 from . import exceptions
 
 # Deprecated
+
 from .topen import topen
 from .stream import Stream as Table
+
+# Version
+
+__version__ = io.open(
```

Comment:
@amercader 
Sorry I meant `setup.py` (upd. above later). 

They (celery, requests) are doing exactly what we're doing - having version somewhere inside the package and then reading it inside the setup.py. So difference only is a question of taste (VERSION vs __init__). 

Our option has been proposed by Paul I suppose and I'm releasing FD packages a lot so it works for me. But still it's far away for solutions like https://github.com/semantic-release/semantic-release